tests/test_ext_inheritance_diagram.py

```diff
@@ -9,9 +9,57 @@
     :license: BSD, see LICENSE for details.
 """
 
-from util import with_app
+import sys
+from util import with_app, rootdir, raises
+from sphinx.ext.inheritance_diagram import InheritanceException, import_classes
 
 
 @with_app('html', testroot='ext-inheritance_diagram')
 def test_inheritance_diagram_html(app, status, warning):
     app.builder.build_all()
+
+
+def test_import_classes():
+    from sphinx.application import Sphinx, TemplateBridge
+    from sphinx.util.i18n import CatalogInfo
+
+    try:
+        sys.path.append(rootdir / 'roots/test-ext-inheritance_diagram')
+        from example.sphinx import DummyClass
+
+        # got exception for unknown class or module
+        raises(InheritanceException, import_classes, 'unknown', None)
+        raises(InheritanceException, import_classes, 'unknown.Unknown', None)
+
+        # a module having no classes
+        classes = import_classes('sphinx', None)
+        assert classes == []
+
+        classes = import_classes('sphinx', 'foo')
+        assert classes == []
+
+        # all of classes in the module
+        classes = import_classes('sphinx.application', None)
+        assert set(classes) == set([Sphinx, TemplateBridge])
+
+        # specified class in the module
+        classes = import_classes('sphinx.application.Sphinx', None)
+        assert classes == [Sphinx]
+
+        # specified class in current module
+        classes = import_classes('Sphinx', 'sphinx.application')
+        assert classes == [Sphinx]
+
+        # ignore current module if name include the module name
```

Comment:
Is this comment correct?
In this case, `sphinx.util.i18n` module will be imported at [sphinx/ext/inheritance_diagram.py L81](https://github.com/sphinx-doc/sphinx/pull/3171/files#diff-502071888c0340c81f31c9f00c7ec5a5R81) , so current module 'sphinx.util' is not ignored.